src/components/Page.tsx

```diff
@@ -123,6 +123,13 @@ class Page extends BaseComponent<PageProps> {
         return false;
       });
     }
+    window.addEventListener("click", event => {
+      if ((event.target as HTMLElement).dataset.linkInternal) {
```

Comment:
I would suggest to use `'linkInternal' in (event.target as HTMLElement).dataset && (event.target as HTMLElement).dataset.linkInternal == true`